GOVERNANCE.adoc

```diff
@@ -0,0 +1,70 @@
+you want to get involved, we ask that you first learn what’s expected of anyone who participates in the project by
+reading the xref:CONTRIBUTING.adoc[Contributor Guidelines].
+
+== Decision-Making and Voting
+ServiceTalk is an open source project, and GitHub serves as the source of truth for the project. Technical disagreements
+may come up, but all community members are always expected to follow the xref:CODE_OF_CONDUCT.adoc[Code of Conduct]. In
+general it is preferred that direction can be mutually agreed upon by community members. If a disagreement cannot be
+resolved independently, the <<Maintainers>> can be called in to vote on the proposal (e.g. issue/PR) in question. An
+independent link:https://github.com/apple/servicetalk/issues[GitHub issues] should be opened prefixed with "[Vote]" and
+reference the proposal in question. The voting period should remain open for at least 2 weeks. Each Maintainer gets one
+vote and the majority will decide the direction. The final decision including the rational for the majority decision
+must be summarized by a Maintainer.
+
+== Maintainers
+The ServiceTalk Maintainers are responsible for the long term maintenance of the ServiceTalk organization. Any
+additions to the repository are weighted on a spectrum of applicability and maintenance overhead. If a feature lies on
+the limited applicability and high maintenance end of the spectrum the maintainers tend to decide against expanding
+scope of the repository. In the future the Maintainers may create a separate organization to facilitate discovery and
+collaboration for community projects. Until this time contributions can be maintained outside the ServiceTalk
+project, demonstrate maturity/adoption, and then be considered for contribution back to the ServiceTalk organization.
+
+=== Become a Maintainer
+Making contributions does not require becoming a maintainer, or obtaining commit access to the ServiceTalk
+organization. However if you are interested in taking a more formal role with decision-making power you should
+consider becoming a Maintainer. Some characteristics of a Maintainer are as follows:
+
+* Be involved in contributing code, pull request review, triage of issues and addressing user questions in one or more
+forums such as GitHub, Stackoverflow, etc.
+* Maintain sustained contribution to the ServiceTalk project and spend a reasonable amount of time on it.
+* Show deep understanding of the areas contributed to, and good consideration of various reliability, usability,
+backward compatibility, and performance requirements.
+
+If you believe you satisfy these above criteria please open a
+link:https://github.com/apple/servicetalk/compare[GitHub pull request] modifying the
+xref:MAINTAINERS.adoc[Maintainers doc] describing at least 5 non-trivial pull requests that have
+been accepted without major modifications. Please also make your case as to why you feel you will need Maintainer status
```

Comment:
I think I would remove the `without major modifications` bit...

Reply:
This is from swift https://swift.org/contributing/#contributing-code. I think it makes sense to keep the bar high and make sure Maintainers can independently drive contributions.

Reply:
+1 to keep it